oui/wireless_usb.sh

```diff
@@ -7,5 +7,6 @@ oui_wireless_usb=(
 	00:25:86='TP-LINK Technologies Co., Ltd.|TP Link TL-WN321G USB Adapter'
 	68:1c:a2='Rosewill Inc.|Rosewill RNX-N150HG USB Adapter'
 	6c:fd:b9='Proware Technologies Co Ltd.|Rosewill RNX-N150HG USB Adapter'
+	c8:3a:35='Ralink Technology, Corp. RT5372 Wireless Adapter'
```

Comment:
Ah yeah just noticed the format, sorry about that. Will fix when I'm back on the PC

-----Original Message-----
From: "Etienne Perot" <notifications@github.com>
Sent: ‎5/‎15/‎2015 0:17
To: "EtiennePerot/macchiato" <macchiato@noreply.github.com>
Cc: "Jeremy Neptune" <jerenept@gmail.com>
Subject: Re: [macchiato] Add Ralink USB Wi-Fi N Adapter OUI (#12)

In oui/wireless_usb.sh:
> @@ -7,5 +7,6 @@ oui_wireless_usb=(
>  	00:25:86='TP-LINK Technologies Co., Ltd.|TP Link TL-WN321G USB Adapter'
>  	68:1c:a2='Rosewill Inc.|Rosewill RNX-N150HG USB Adapter'
>  	6c:fd:b9='Proware Technologies Co Ltd.|Rosewill RNX-N150HG USB Adapter'
> +	c8:3a:35='Ralink Technology, Corp. RT5372 Wireless Adapter'
This doesn't have the right format. Please see https://github.com/EtiennePerot/macchiato/blob/master/README.md#contribute (last paragraph).
—
Reply to this email directly or view it on GitHub.